app/src/main/res/layout/activity_login.xml

```diff
@@ -1,35 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
+    android:layout_height="wrap_content"
     android:layout_margin="16dp"
     android:orientation="vertical"
-    tools:context=".LoginActivity">
+    tools:context=".LoginActivity" >
+
+    <ImageView
+        android:id="@+id/ivLogo"
+        android:layout_width="match_parent"
+        android:layout_height="75dp"
+        android:src="@drawable/nav_logo_whiteout"
+        android:layout_marginTop="100dp"/>
 
     <EditText
         android:id="@+id/etUsername"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:ems="10"
-        android:hint="username" />
+        android:layout_height="45dp"
+        android:hint="Username"
+        android:layout_below="@+id/ivLogo"
+        android:background="@drawable/stroke_box"
+        android:layout_marginTop="5dp"
+        android:paddingStart="15dp"/>
 
     <EditText
         android:id="@+id/etPassword"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_weight="1"
+        android:layout_height="45dp"
         android:ems="10"
-        android:hint="password"
-        android:inputType="textPassword" />
+        android:hint="Password"
+        android:inputType="textPassword"
+        android:layout_below="@+id/etUsername"
+        android:background="@drawable/stroke_box"
+        android:layout_marginTop="10dp"
+        android:paddingStart="15dp"/>
 
     <Button
         android:id="@+id/btnLogin"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:text="Login" />
+        android:text="Log in"
+        android:textAllCaps="false"
+        android:textSize="18sp"
+        android:textColor="@android:color/white"
+        android:backgroundTint="#458eff"
```

Comment:
I'd suggest storing this color to your style.xml file in case you need to use it again in the future. It's also going to be easier for reader to somehow imagine your layout